keyboards/nack/keymaps/default/keymap.c

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2020 pastapojken <pastapojken@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "keymap.h"
+
+#define ____ KC_TRNS
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```

Reply:
Technically I'm defining that 4 underscores are transparent. It's only there to give a better aesthetic in the keymap, I've made that clearer now with:
`#define ____ _______`
I can make it conform to the standard otherwise :)

Reply:
I can understand the concerns about aesthetic.
On the other hand, it can be confusing for others, especially for new users.
`____` works, `_______` works, but `______` does not, neither does `_____`

Personally, I don't have a problem with such stuff in personal keymaps, but `default` is the base others will use to start working on their own stuff and as such, the less is more.

Reply:
Good point, I'll fix it.